src/pages/LoginPage.js

```diff
@@ -33,35 +36,44 @@ function Login() {
   }, [navigate]);
 
   return (
-    <div className="LoginPage">
-      <div className="container">
+    <div className="LoginPage d-flex  justify-content-center align-items-center">
+      <Container className=" d-flex justify-content-between bg-white rounded-4">
         <div className="right">
-          <h2 className="h2title">Welcome to To Do!</h2>
-          <p className="ptitle">
+          <h2 className="h2title mt-4">Welcome to To Do!</h2>
+          <p className="mt-3">
             "Manage your time, organize your tasks with To Do!"
           </p>
-          <h1 className="h1title">Log In</h1>
+          <h1 className="mt-3 text-dark">Log In</h1>
```

Comment:
This part should be in a semantic element like section or summary.
https://www.w3schools.com/html/html5_semantic_elements.asp